torch_em/segmentation.py

```diff
@@ -253,6 +253,7 @@ def default_segmentation_dataset(
     with_channels=False,
     with_label_channels=False,
     verify_paths=True,
+    with_padding_per_patch=False,
```

Comment:
Update the name (see above), set default to True and pass it to ImageCollectionDataset as well (after adding it there as well).